tutorial-running-chaos-workflow-on-remote-cluster/index.html

```diff
@@ -308,8 +308,8 @@ <h3 is-upgraded><strong>Continue?</strong></h3>
 role.rbac.authorization.k8s.io/argo-role created
 rolebinding.rbac.authorization.k8s.io/argo-binding created
 
-💡Connecting agent to Chaos Center.
-💡Connecting agent to Chaos Center.
+💡Connecting agent to ChaosCenter.
```

Comment:
Is this what we also see in litmusctl today

Reply:
No, I guess we would have to update the commands as well.

Reply:
Updated, can you PTAL now?